feat(insights): add note to clarify difference in webvital thresholds between google and sentry (#10552)
getsentry/sentry-docs: docs/product/insights/web-vitals/index.mdx

| [Time To First Byte](/product/insights/web-vitals/web-vitals-concepts/#time-to-first-byte-ttfb) (TTFB)           | [200ms](https://www.desmos.com/calculator/ykzahw9goi)  | [400ms](https://www.desmos.com/calculator/ykzahw9goi)  | ~14%   |

<Note>

> These thresholds are set to match Google Lighthouse as close as possible, but there may be differences since Sentry operates on real user data, whereas Google Lighthouse operates on lab data.

Reviewer: ```suggestion
These thresholds are meant to match Google Lighthouse as closely as possible, but there may be some differences because Sentry operates on real user data, whereas Google Lighthouse operates on lab data.
```
